### DISCUSSION TITLE
Osu!lazer has a problem

### DISCUSSION THREAD

**Discussion by Adrik0123** [ppy/osu#24094] [Q&A]
When I want to play Osu!lazer the app close and say ' Osu!lazer has stopped and I verify and I have a adroid 5.0.1 so it may work no?

**Comment by integerrr** [comment]
#15188 please

**Reply by Adrik0123** [reply]
Please what

**Reply by monochrome22** [reply]
Provide logs.
How to do so is explained in the link (https://github.com/ppy/osu/discussions/15188).

**Comment by MineTurtlee** [comment]
> When I want to play Osu!lazer the app close and say ' Osu!lazer has stopped and I verify and I have a adroid 5.0.1 so it may work no?

Uhm, it works from Android 5.0 so it is not **IMPOSSIBLE** to say that it's unsupported...
